api/python/quilt3/util.py

```diff
@@ -520,15 +520,19 @@ def catalog_s3_url(catalog_url, s3_url):
     return url
 
 
-def catalog_package_url(catalog_url, bucket, package_name, package_timestamp="latest"):
+def catalog_package_url(catalog_url, bucket, package_name, package_timestamp="latest", tree=True):
```

Comment:
we don't need to handle the `tree=True` case in this method at all because the tree is not part of the packages (sending users to/tree was a temporary hack) and therefore doesn't belong in catalog_package_url

Reply:
Meh, I can live with this for now